types/sawtooth.d.ts

```diff
@@ -150,15 +150,12 @@ declare namespace Sawtooth {
       setState<T extends Record<string, Buffer>, U extends keyof T>(
         state: T,
         timeout?: number,
-      ): Promise<[U]>;
+      ): Promise<U[]>;
```

Comment:
?

Reply:
Fixed copy pasta from sawtooth. Don't worry about it